pkg/utils/nodeclass/suite_test.go

```diff
@@ -247,6 +247,74 @@ var _ = Describe("NodeClassUtils", func() {
 		ExpectSecurityGroupStatusEqual(nodeTemplate.Status.SecurityGroups, nodeClass.Status.SecurityGroups)
 		ExpectAMIStatusEqual(nodeTemplate.Status.AMIs, nodeClass.Status.AMIs)
 	})
+	It("should convert a AWSNodeTemplate to a EC2NodeClass (with AMISelector name and owner values set) with spaces", func() {
```

Comment:
Do we have a test for cases where there are spaces in the name itself, similar to `"aws::owners": "self, my     owner, 123456789"`